docs/pages/api-docs/autocomplete.md

```diff
@@ -46,7 +46,7 @@ The `MuiAutocomplete` name can be used for providing [default props](/customizat
 | <span class="prop-name">disabled</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the input will be disabled. |
 | <span class="prop-name">disabledItemsFocusable</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, will allow focus on disabled items. |
 | <span class="prop-name">disableListWrap</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the list box in the popup will not wrap focus. |
-| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |
+| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | The `children` will be children in the DOM. |
```

Comment:
Can we have another round on this one? cc @joshwooding

Reply:
Hmm, the previous description was better. 
Maybe something like:
The `PaperComponent` will be inside the DOM hierarchy of the parent component.

for the autocomplete?

Reply:
If we can remove all context to the current component, it would be better, this label is shared.

Reply:
Could we swap it with '`Popper` content'?

Reply:
Ok

Reply:
So: 

The `Popper` content will be inside the DOM hierarchy of the parent component. 

?